packages/sanity/src/core/form/studio/tree-editing/components/search/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const ITEM_HEIGHT = 33
```

Comment:
Do we need this? 🤔

Reply:
Yes, it is used calculate the scroll height for virtualization + set the max height of the popover